Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -681,6 +687,7 @@ func SimpleKubelet(client *client.Client,
 		ResolverConfig:      kubelet.ResolvConfDefault,
 		ResourceContainer:   "/kubelet",
 		RootDirectory:       rootDir,
+		SerializeImagePulls: true,
```

Comment:
ok - this was a change between 3.0.2 backport and now.

https://github.com/openshift/ose/pull/94/files#diff-b1cd32a6f03a1a0a5f5c1b0994cc3eeeR363

I will have to add a follow-on commit to enable it.